corehq/apps/hqadmin/service_checks.py

```diff
@@ -123,7 +123,11 @@ def check_kafka():
 
 @change_log_level('urllib3.connectionpool', logging.WARNING)
 def check_elasticsearch():
-    cluster_health = check_es_cluster_health()
+    try:
+        cluster_health = check_es_cluster_health()
+    except Exception:
+        return ServiceStatus(False, "Something went wrong checking cluster health")
```

Comment:
> adding this to at least better pinpoint where the error is coming from

This seems like it will mask the deeper error with another generic error message. Maybe print a stack trace somewhere or send it to sentry? I think I'm not understanding how this will help _pinpoint_ where it's coming from.

Reply:
Yeah good point. It would only confirm that the error is coming out of this code, but then followup steps would be needed. Sending to sentry sounds ideal.

Reply:
8072d469cf260038e2b4ae92694efa2ef903a732